CHANGELOG.md

```diff
@@ -7,9 +7,9 @@ and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ## Unreleased
 
-### Fixed
+### [3.7.3] - 2025-01-30
```

Comment:
Also, please bump the version in `Cargo.toml`

Reply:
thanks, done in https://github.com/paritytech/parity-scale-codec/pull/690/commits/8f96f0cc289d5fdb05c166665b81cf0cd69abce8